tests/fake_messages/fedora-ci-message-brew-build.test.complete-2.0.0.json

```diff
@@ -1,10 +1,11 @@
 {
   "body": {
     "msg": {
-      "ci": {
+      "contact": {
```

Comment:
Is there a test that covers the fallback case?

Reply:
It's not specifically tested, but other fixtures use "ci". Not sure how to strongly we need to ensure compatibility since the version of the message is <1.0.0.